back/app/services/admin_publications_filtering_service.rb

```diff
@@ -5,6 +5,8 @@ class AdminPublicationsFilteringService
 
   # NOTE: This service is very fragile and the ORDER of filters matters for the Front-End, do not change it.
 
+  project_publications = nil
```

Comment:
I'm not sure this is ideal. Suggestions to improve appreciated!

Because project_publications are found in the `'filter_projects'` filter, and I want to make use of this collection again in the `'remove_parents_with_only_not_allowed_children'` filter I add in this PR, I define this var up front to keep it available in all filter scopes ('scope' as in block-level/namespace scope).